src/sentry/snuba/metrics/naming_layer/mapping.py

```diff
@@ -70,9 +70,11 @@ def get_public_name_from_mri(internal_name: Union[TransactionMRI, SessionMRI, st
 
 
 def get_operation_with_public_name(operation: Optional[str], metric_mri: str) -> str:
-    if operation is None:
-        return get_public_name_from_mri(metric_mri)
-    return f"{operation}({get_public_name_from_mri(metric_mri)})"
+    return (
+        f"{operation}({get_public_name_from_mri(metric_mri)})"
```

Comment:
is this change unrelated? if so let's remove it.

Reply:
It is related yes